api/specs.proto

```diff
@@ -241,3 +241,15 @@ message ClusterSpec {
 	// CAConfig defines cluster-level certificate authority settings.
 	CAConfig ca_config = 6 [(gogoproto.nullable) = false, (gogoproto.customname) = "CAConfig"];
 }
+
+// ContainerAttachmentSpec specifies the allocation information
+// for the container attachment.
+message ContainerAttachmentSpec {
+	Annotations annotations = 1 [(gogoproto.nullable) = false];
+	
```

Comment:
The container is not needed. From worker point of view, he needs the manager to allocate the IP resources needed to connect an entity to the network.